backend/dev/src/dev.clj

```diff
@@ -2,32 +2,53 @@
   (:refer-clojure :exclude [test])
   (:require [akvo.lumen.endpoint]
             [akvo.lumen.lib.aes :as aes]
+            [akvo.lumen.middleware]
             [akvo.lumen.migrate :as lumen-migrate]
             [clojure.edn :as edn]
             [clojure.java.io :as io]
             [clojure.java.jdbc :as jdbc]
             [clojure.pprint :refer [pprint]]
             [clojure.repl :refer :all]
             [clojure.tools.namespace.repl :refer [refresh]]
-            [com.stuartsierra.component :as component]
+            [duct.core :as duct]
             [duct.generate :as gen]
-            [duct.util.repl :refer [setup test cljs-repl] :as duct-repl]
-            [duct.util.system :refer [load-system]]
-            [reloaded.repl :refer [system init start stop go reset]])
+            [integrant.core :as ig]
+            [integrant.repl :as ir]
+            [integrant.repl.state :as state :refer (system)])
   (:import [org.postgresql.util PSQLException PGobject]))
 
-(defn new-system []
-  (load-system (keep io/resource
-                     ["akvo/lumen/system.edn" "dev.edn" "local.edn"])))
+(defn read-config []
+  (duct/read-config (io/resource "dev.edn")))
+
+(derive :akvo.lumen.component.emailer/dev-emailer :akvo.lumen.component.emailer/emailer)
+(derive :akvo.lumen.component.caddisfly/local :akvo.lumen.component.caddisfly/caddisfly)
+(derive :akvo.lumen.component.error-tracker/local :akvo.lumen.component.error-tracker/error-tracker)
+
+(defn dissoc-prod-components [c]
```

Comment:
Not sure why are directly referencing the production one and then removing it at dev time, instead we should reference the generic *parent* one?

https://github.com/weavejester/integrant#derived-keywords

Reply:
After talking with @iperdomo we agree on ignoring this comment 
(Expanded bits are that integrant needs the implementation (child) keyword in dev.edn or config.edn thus integrant will search for the init-key defmethod for the generic (parent) keyword component to be initiated)